src/note-components/Label/label.jsx

```diff
@@ -0,0 +1,52 @@
+import { Navbar } from "../Navbar/navbar";
+import { Sidebar } from "../Sidebar/sidebar";
```

Comment:
Better to use use index.js file in the same directory to import export these components

Reply:
Okay